__tests__/app.test.js

```diff
@@ -90,6 +90,23 @@ describe('/api/articles/:article_id', () => {
             expect(message).toBe('Bad Request')
         })
     })
+    test('GET 200: Should return the article with a comment_count property added on', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({ body: { article }}) => {
+            expect(article).toHaveProperty('comment_count')
+        })
+    })
+    test('GET 200: Should return the article with a comment_count of 0 if provided an article with no comments', () => {
+        return request(app)
+        .get('/api/articles/2')
+        .expect(200)
+        .then(({ body: { article }}) => {
+            expect(article).toHaveProperty('comment_count')
+            expect(article.comment_count).toBe(0)
+        })
+    })
```

Comment:
Great tests Billy, and because you made your previous tests flexible, you didn't have to change them!